src/main/setupInvestHandlers.js

```diff
@@ -143,11 +143,16 @@ export function setupInvestRunHandlers(investExe) {
     // in which case it's useful to logger.debug too.
     investRun.stderr.on('data', (data) => {
       logger.debug(`${data}`);
-      event.reply(`invest-stderr-${channel}`, `${data}${os.EOL}`);
+      // The PyInstaller exe will always emit a final 'Failed ...' message
+      // after an uncaught exception. It is not helpful to display to users
+      // so we filter it out here.
+      const dat = `${data}`
+        .split(os.EOL)
+        .filter((line) => !line.match(/\[[0-9]+\] Failed to execute script/))
```

Comment:
So this just removes the PyInstaller `Failed` message from any stderr we want to report back?

Reply:
That's it!